bittensor/btlogging/loggingmachine.py

```diff
@@ -70,19 +70,19 @@ class LoggingMachine(StateMachine):
         | Default.to(Default)
     )
 
-    enable_trace: Trace = (
+    enable_trace = (
         Default.to(Trace) | Debug.to(Trace) | Disabled.to(Trace) | Trace.to(Trace)
     )
 
-    enable_debug: Debug = (
+    enable_debug = (
         Default.to(Debug) | Trace.to(Debug) | Disabled.to(Debug) | Debug.to(Debug)
     )
 
-    disable_trace: Default = Trace.to(Default)
+    disable_trace = Trace.to(Default)
 
-    disable_debug: Default = Debug.to(Default)
+    disable_debug = Debug.to(Default)
 
-    disable_logging: Disabled = (
+    disable_logging = (
```

Comment:
Why are we removing type hints?

Reply:
I think the type hints here are not required as, for example, 'Debug' is not a type here; instead it is an instance of the State class in the Statemachine. 
The IDE also warns with the error
`Variable not allowed in type expression`